### PR TITLE
move theme switcher to sidebar and replace with +

### LINKED ISSUE #927: Move theme switcher to sidebar, replace by +
Theme switcher should move to side bar like shown on roadmap. 

This gives us room to have a consistent + button on both the MyGoals and the MyTime pages.

When pressing +, can we switch back to MyGoals before showing the dropdown options?

### PR DESCRIPTION
**Pull request by elyessAR**
Closes #927.
Moved theme switcher to sidebar and replaced with +, user will be redirected to myGoals page and "add new goal" will open up" when + pressed from My time page.

### CODE REVIEW

File: src/components/HeaderDashboard/HeaderDashboard.scss [hunk 1]
```diff
@@ -45,9 +45,8 @@
 }
 
 .dark-mode-toggle {
-  position: absolute;
-  top: 25px;
-  right: 18px;
+  margin-left: 138px;
```
Comment (Tushar-4781) [inline src/components/HeaderDashboard/HeaderDashboard.scss:48]: Not a good way to position an element to center.

**Review by Tushar-4781 — COMMENTED** [review]
Try something better, the toggler is not in center and in some devices it is shifting to right

**Comment by elyessAR** [comment]
is this better @Tushar-4781 
Screenshot_4
Screenshot_5

**Comment by Tushar-4781** [comment]
> is this better ?@Tushar-4781 Screenshot_4 Screenshot_5

without commit i can't say anything. but yes the ss u sent is correct

**Comment by elyessAR** [comment]
> > is this better ?@Tushar-4781 Screenshot_4 Screenshot_5
> 
> without commit i can't say anything. but yes the ss u sent is correct

Ok, glad you liked it I'll commit

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
image

Can you add some margin between options div and toggler

**Comment by elyessAR** [comment]
> image
> 
> Can you add some margin between options div and toggler

Sure

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**